spec/student_spec.rb

```diff
@@ -0,0 +1,21 @@
+require_relative './spec_helper'
+
+describe Student do
+  student = Student.new('C1', 22, name: 'Lucca')
```

Comment:
- [ ] I humbly suggest that you follow different suggestions that I added and apply them throughout your spec files so that tests run successfully because there needs to be consistency so that the other error I showed which was appearing when I tried to run tests, is solved and therefore tests can be run well 😃 👍